packages/next-css/index.js

```diff
@@ -23,14 +23,6 @@ module.exports = (nextConfig = {}) => {
 
       config.module.rules.push({
         test: /\.css$/,
-        issuer(issuer) {
```

Comment:
This issuer check needs to happen - its a css ordering problem. Global css needs to come from the _app - so ordering can be applied to it. And downstream css imports can have its index ignored.

Reply:
Thanks I’ll revise